tests/views/test_accounts.py

```diff
@@ -14,13 +14,21 @@
 project_root = os.path.dirname(os.path.dirname(__file__))
 with open(f"{project_root}/test_data/uaa/user_by_id.json") as json_data:
     uaa_user_by_id_json = json.load(json_data)
+with open(f"{project_root}/test_data/uaa/updated_name.json") as json_data:
+    updated_name_json = json.load(json_data)
 
 test_email = "fake@ons.gov.uk"
 user_id = "fe2dc842-b3b3-4647-8317-858dab82ab94"
 url_uaa_token = f"{TestingConfig.UAA_SERVICE_URL}/oauth/token"
 url_uaa_get_accounts = f"{TestingConfig.UAA_SERVICE_URL}/Users?filter=email+eq+%22{test_email}%22"
 url_uaa_get_user_by_id = f"{TestingConfig.UAA_SERVICE_URL}/Users/{user_id}"
 url_uaa_create_account = f"{TestingConfig.UAA_SERVICE_URL}/Users"
+url_uaa_update_name = f"{TestingConfig.UAA_SERVICE_URL}/Users/{user_id}"
```

Comment:
instead of adding another one, remove this and make the one above have a more generic name like `url_uaa_user_by_id` then have everything use that instead